template.yml

```diff
@@ -1,6 +1,31 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: Sends log data from S3 to New Relic Logging.
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: NewRelic-log-ingestion-s3-RPB
```

Comment:
Is there any reason to change the application name?

Reply:
thanks. I was testing my own copy of the application - where I changed the application name. 
The application name should remain for the official release.